### DISCUSSION TITLE
Discord へ移行します

### DISCUSSION THREAD

**Discussion by kikuchiharuma** [project-theory-probe/ptp-voyage#15] [Announcements]
Journal 2 まで Discussions でのやりとりを試しましたが、ちょっとスピード感に欠けたため Discord へ移行します
↓こちら

Project Theory Probe Discord Community 
https://discord.gg/9ndJ7x7x

**Comment by motoi** [comment]
リンクが切れていると表示されます！

**Comment by kikuchiharuma** [comment]
すいません！
有効なものに変えました！
